src/Sulu/Bundle/WebsiteBundle/Cache/CacheClearer.php

```diff
@@ -80,7 +87,10 @@ public function clear(/*?array $tags = null*/)
 
         $tags = \func_num_args() >= 1 ? \func_get_arg(0) : null;
 
-        if (null !== $tags && $this->cacheManager && $this->cacheManager->supportsTags()) {
+        dump($this->tagsEnabled);
+        die;
+
+        if (null !== $tags && $this->cacheManager && $this->cacheManager->supportsTags() && $this->tagsEnabled) {
```

Comment:
```suggestion
        if (null !== $tags && $this->tagsEnabled && $this->cacheManager && $this->cacheManager->supportsTags()) {
```

would check the variable before checking the service. so it avoids the service call.